src/components/common/needsShutdown.tsx

```diff
@@ -182,7 +190,8 @@ export function getDevicesRequiringShutdown(vm: VM) {
 
     // DISKS
     for (const target in vm.disks) {
-        if (needsShutdownDiskAccess(vm, target)) {
+        if (needsShutdownDiskAccess(vm, target) ||
+            needsShutdownDiskCache(vm, target)) {
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2466-9900a690-20251229-144551-fedora-43-devel/Coverage/lcov/github-pr.diff.gcov.html)